packages/components-css/side-navigation/index.scss

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2024 Community for NL Design System
+ */
+
+.kernteam-side-navigation {
+  border-inline-end-color: var(--kernteam-side-navigation-border-inline-end-color);
+  border-inline-end-style: var(--kernteam-side-navigation-border-inline-end-style);
+  border-inline-end-width: var(--kernteam-side-navigation-border-inline-end-width);
+  display: flex;
+  flex-direction: column;
+  inline-size: var(--kernteam-side-navigation-inline-size);
+  transition: inline-size 200ms ease;
+}
+
+.kernteam-side-navigation--hidden {
+  inline-size: 48px;
+}
+
+.kernteam-side-navigation--hidden .kernteam-side-navigation__content {
+  overflow-x: hidden;
+  visibility: hidden;
+}
+
+.kernteam-side-navigation-nav {
+  flex-grow: 1;
+}
+
+.kernteam-side-navigation__toggle-button {
+  padding-inline-start: var(--kernteam-side-navigation-toggle-button-padding-inline-start);
+}
+
+.kernteam-menu-list {
+  --utrecht-link-color: var(--kernteam-menu-list-link-color);
+  --utrecht-link-text-decoration: none;
+  --utrecht-button-subtle-color: var(--utrecht-link-color);
+  --utrecht-button-focus-color: var(--kernteam-focus-color);
+  --utrecht-button-icon-size: 28px;
+
+  display: block;
+  list-style: none;
+  padding-block-end: 0;
+  padding-block-start: 0;
+  padding-inline-end: 0;
+  padding-inline-start: 0;
+}
+
+.kernteam-menu-list-item__label {
+  align-items: center;
+  display: inline-flex;
+  inline-size: 100%;
+}
+
+.kernteam-menu-list-item__label:hover {
+  background-color: var(--kernteam-menu-list-link-hover-background-color);
+}
+
+.kernteam-menu-list__icon-button {
+  cursor: pointer;
+}
+
+.kernteam-menu-list-item__link {
+  display: flex;
+  flex: 1;
+  line-height: 1.5;
+  margin-inline-start: 4px;
+  padding-block: 0.75rem;
```

Comment:
hier kunnen we shorthand gebruiken

Reply:
Als de stylelint in te stellen is om non-logical-shorthands zoals padding alleen toe te staan met 1 length, dan is het okay.

Anders is het handiger om padding volledig te blokkeren.